src/main/kotlin/com/cognifide/gradle/aem/deploy/responses/ListResponse.kt

```diff
@@ -1,4 +1,4 @@
-package com.cognifide.gradle.aem.deploy
+package com.cognifide.gradle.aem.deploy.responses
```

Comment:
After discussion, we agreed that the best solution would be leaving package structure as it was before refactoring.